jupyterlab_git/git.py

```diff
@@ -16,6 +16,45 @@ def __init__(self, root_dir, *args, **kwargs):
         super(Git, self).__init__(*args, **kwargs)
         self.root_dir = os.path.realpath(os.path.expanduser(root_dir))
 
+    def config(self, **kwargs):
+        """Get or set Git options.
+        
+        If no kwargs, all options are returned. Otherwise kwargs are set.
+        """
+        response = {"code": 1}
+
+        if len(kwargs):
+            output = []
+            for k, v in kwargs.items():
+                cmd = ["git", "config", "--global", "--add", k, v]
```

Comment:
In my opinion, setting these are the global level and potentially overwriting global configuration is overkill. As long as we set it at the repo level, it should suddice.

Reply:
I set options at repository level